pkg/validation/app.go

```diff
@@ -19,6 +19,7 @@ const (
 	catalogNotFoundTemplate         = "catalog %#q not found"
 	nameTooLongTemplate             = "name %#q is %d chars and exceeds max length of %d chars"
 	nameNotFoundReasonTemplate      = "name is not specified for %s"
+	targetNamespaceNotAllowed       = "target namespace %s is not allowed"
```

Comment:
```suggestion
	targetNamespaceNotAllowed       = "target namespace %s is not allowed for in-cluster apps"
```

We should tell the user why the change is rejected.

Reply:
Good point, done